README.md

```diff
@@ -1,3 +1,5 @@
+![Mobius Logo](https://github.com/spotify/mobius/wiki/mobius-logo.png)
```

Comment:
Awesome :) How about you use it to replace the `Mobius` heading below instead, so we don't have a double header?

Reply:
Should I do that in both this repo and the Swift one?

Reply:
Sounds reasonable, if they have the same README structure